src/app/app.component.ts

```diff
@@ -74,6 +74,7 @@ export class AppComponent implements OnInit {
     private windowRef: SkyAppWindowRef,
     private config: SkyAppConfig,
     private styleLoader: SkyAppStyleLoader,
+    private helpInitService: HelpInitializationService,
```

Comment:
Could we use a similar approach to the optional `searchProvider` below this line, and mark the service as optional + check for it's existence for using it? I believe this would allow us to conditionally inject it in `lib/sky-pages-module-generator.js`, thereby not including the library for SPAs that don't have `help` defined in `skyuxconfig.json`.  Thoughts?

Reply:
Good thoughts! I'll see what I can do.